src/resources/UsageAnalytics/Read/Exports/ExportsInterfaces.ts

```diff
@@ -87,6 +87,37 @@ export interface GenerateExportParams extends EstimateExportParams {
     useDisplayNames?: boolean;
 }
 
+type ExportFormat = 'EXCEL' | 'NO_NEWLINE';
+export type ExportTable = 'SEARCHES' | 'CLICKS' | 'CUSTOM_EVENTS' | 'KEYWORDS' | 'GROUPS';
+
+export interface GenerateExportWithBodyParams {
```

Comment:
Several properties are duplicated between the two interfaces, you could create a base interface and have the other two extend it:

```typescript
interface BaseGenerateExportBodyParams {
  from: string;
  to: string;
  fileName?: string;
  description?: string;
  dimensions?: string[];
  exportFormat?: ExportFormat;
  useDisplayName?: boolean;
}

export interface GenerateExportWithBodyParams extends BaseGenerateExportBodyParams {
  commonFilters?: string[];
  searchesFilters?: string[];
  customEventsFilters?: string[];
  tables?: ExportTable[];
}

export interface GenerateVisitExportWithBodyParams extends BaseGenerateExportBodyParams {
  inclusionFilters?: string[];
  hideEventFilters?: string[];
  exclusionFilters?: string[];
  ordered?: boolean;
}
```

Reply:
Sure :thumbsup: I can also copy the Swagger annotations and document these better.